tests/unit/fetcher-test.js

```diff
@@ -238,39 +267,51 @@ describe('Fetcher', () => {
 
       fetcher = new Fetcher(rdf.graph())
 
+      // No, this is internal function!
       options = fetcher.initFetchOptions(uri, {})
+      nock('https://example.com').get('/doc.ttl').reply(400, errorMessage) // sets body not statusText
+      // options = {}
 
       statusCode = 400
     })
 
-    it('should return a result object', () => {
-      return fetcher.failFetch(options, errorMessage, statusCode)
-        .then(result => {
-          expect(result.ok).to.be.false()
-          expect(result.status).to.equal(statusCode)
-          expect(result.error).to.equal(errorMessage)
+    it('should return a result object on HTTP error', () => {
+      return fetcher.load(uri, options)
+        .then(result => {}, err => {
+          console.log('@@@@@@ err is ' + err)
+          console.log('@@@@@@ err.response is ' + err.response)
+          console.log('@@@@@@ err.response.status is ' + err.response.status)
+          console.log('@@@@@@ err.response.statusText is ' + err.response.statusText)
```

Comment:
Tests instead of logs?

Reply:
This was debugging a test.  Removed.  The test was wrong, in that it tried to nock() the statusText using .reply(status, statusText)  but the API is actually .reply(status, body).  So can't check that the statusText comes through by injecting one.